app/routes/goal_routes.py

```diff
@@ -0,0 +1,97 @@
+from flask import Blueprint, request, jsonify
+from app.models.goal import Goal
+from app import db
+from app.routes.routes_helper_function import handle_valid_id
+from app.models.task import Task
+
+#Blueprint for goals, all routes have url prefix (/goals)
+goals_bp = Blueprint("goals_bp", __name__, url_prefix="/goals")
+
+@goals_bp.route("", methods=["POST"])
+def create_goal():
+    request_body = request.get_json()
+    if 'title' not in request_body:
+        return {"details": "Invalid data"}, 400
+    
+    new_goal = Goal(title=request_body["title"])
+
+    db.session.add(new_goal)
+    db.session.commit()
```

Comment:
Great job using `db.session.<method-name>.` Session gives us access to the following:
* `db` is our app’s instance of SQLAlchemy.
* `session` represents our active database connection.
* By referencing db.session we can use SQLAlchemy’s methods to perform tasks like committing a change to a model, storing a new record of a model, and deleting a record.

By referencing `db.session.add()` you are able to use the SQLAlchemy method to store a new record of the Goal model. ⭐